src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SingleSelection.js

```diff
@@ -150,14 +157,24 @@ export default class SingleSelection extends React.Component<Props>
                 ...formListOptions,
                 ...typeOptions,
             }
-            : undefined;
+            : {};
+
+        if (requestParameters) {
```

Comment:
check should not be needed. and you should be able use a single `forEach` instead of looping through the parameters two times 🙂

Reply:
i will merge the `forEach` but the if is neccesary because of flow 😂

Reply:
okay 🙈